pkg/bff/server.go

```diff
@@ -363,6 +363,7 @@ func (s *Server) setupRoutes() (err error) {
 
 		// GDS public routes (no authentication required)
 		v1.GET("/lookup", s.Lookup)
+		v1.GET("/vasps", s.VASPNames)
```

Comment:
I wonder if we should add this at `/lookup/autocomplete` instead -- since that's more specific to what we're using this list for. It is possible that we may need a `/vasps` endpoint in the future for a read-only REST endpoint with List and Detail.

Reply:
Sure, makes sense to me.